_rules/links-identical-name-equivalent-purpose-b20e66.md

```diff
@@ -11,7 +11,7 @@ accessibility_requirements:
     passed: further testing needed
     inapplicable: further testing needed
 wcag20:2.4.4: # Link Purpose (In Context) (A)
-secondary: true
+secondary: This success criterion is **less strict** than this rule. This is because the rule does not consider the context of the link. This is why some of the failed examples may not satisfy this success criterion.
```

Comment:
If the success criterion is less strict, shouldn't the failed examples potentially satisfy the criterion (e.g., where the link text does not provide the purpose but the context does).

Reply:
In regard to @tbostic32's concerns with the over-use of the word "this"... 
Without commenting on the accuracy of the statements, a rewrite of the prior secondary could be:

>This success criterion is **less strict** than this rule because the rule does not consider the context of the link. Therefore, some of the failed examples may not satisfy this success criterion.

@tbostic32's second comment seems correct to me just reading the sentence statements. If the following is not accurate, then I think there's a larger problem here...

>This success criterion is **less strict** than this rule because the rule does not consider the context of the link. Therefore, some of the failed examples may ~not~ satisfy this success criterion.